---
title: "R tools for microbiome research"
output: html_document
---



> A   

Adaptive gPCA [A method for structured dimensionality reduction](https://arxiv.org/abs/1702.00501)     
Ampvis2 [Tools for visualising amplicon sequencing data](https://madsalbertsen.github.io/ampvis2/)   
ANCOM/ANCOM-BC [R package for Analysis of Composition of Microbiomes (ANCOM-BC)](https://github.com/FrederickHuangLin/ANCOMBC)   
animalcules [R shiny app for interactive microbiome analysis](https://compbiomed.github.io/animalcules-docs/)   
ALDEx2 [Analysis Of Differential Abundance Taking Sample Variation Into Account](https://bioconductor.org/packages/release/bioc/html/ALDEx2.html)  


> B    

BDMMA [Batch Effects Correction for Microbiome Data With Dirichlet-multinomial Regression](https://academic.oup.com/bioinformatics/article-abstract/35/5/807/5078473)   
BEEM [BEEM: Estimating Lotka-Volterra models from time-course microbiome sequencing data](https://github.com/lch14forever/BEEM)   
biome-shiny [GUI for microbiome visualization, based on the shiny package "microbiome"](https://github.com/Biodata-PT/Biome-Shiny)   
bootLong [The Block Bootstrap Method for Longitudinal Microbiome Data](https://pratheepaj.github.io/bootLong/)   
breakaway [Species Richness Estimation and Modeling](https://github.com/search?q=breakaway)   

> C   

CCREPE [Compositionality Corrected by PErmutation and REnormalization](http://bioconductor.org/packages/release/bioc/html/ccrepe.html)   
corncob [Modeling microbial abundances and dysbiosis with beta-binomial regression](https://arxiv.org/abs/1902.02776)  
curatedMetagenomicData [Accessible, curated metagenomic data through ExperimentHub](https://waldronlab.io/curatedMetagenomicData/)    
CoDaSeq [Functions for compositional analysis of high throughput sequencing](https://github.com/ggloor/CoDaSeq)     


> D     

dacomp [Testing for Differential Abundance in Compositional Counts Data, with Application to Microbiome Studies](https://github.com/barakbri/dacomp)   
DADA2 [Divisive Amplicon Denoising Algorithm](https://www.nature.com/nmeth/journal/v13/n7/full/nmeth.3869.html)    
DECIPHER [Using DECIPHER v2.0 to Analyze Big Biological Sequence Data in R](http://www2.decipher.codes/index.html)   
DECIPHER/IIDTAXA [IDTAXA: a novel approach for accurate taxonomic classification of microbiome sequences](doi:10.1186/s40168-018-0521-5)   
decontam [Simple Statistical Identification and Removal of Contaminant Sequences in Marker-Gene and Metagenomics Data](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0605-2)   
DESeq2 [Differential expression analysis for sequence count data](https://www.bioconductor.org/packages/devel/bioc/vignettes/phyloseq/inst/doc/phyloseq-mixture-models.html)   
DivNet [An R package to estimate diversity when taxa in the community cooccur via a ecological network](https://github.com/adw96/DivNet)  
DMBC [A Dirichlet-Multinomial Bayes Classifier for Disease Diagnosis With Microbial Compositions](https://msphere.asm.org/content/2/6/e00536-17)    

> E   

edgeR [empirical analysis of DGE in R](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2796818/)    

> G     

GLMMMiRKAT [A distance-based kernel association test based on the generalized linear mixed model](https://github.com/hk1785/GLMM-MiRKAT)   

> I   

igraph [Network Analysis and Visualization in R](http://igraph.org/r/)    

> L    

labdsv [Ordination and Multivariate Analysis for Ecology](https://cran.r-project.org/web/packages/labdsv/labdsv.pdf)  

> M   

MaAsLin2 [MaAsLin2: Microbiome Multivariate Association with Linear Models](https://github.com/biobakery/Maaslin2)   
mare [Microbiota Analysis in R Easily](https://github.com/katrikorpela/mare)   
massMap [A Two-Stage Microbial Association Mapping Framework With Advanced FDR Control](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-018-0517-1)  
mbtools [Collection of R tools to analyze microbiome data](https://gibbons-lab.github.io/mbtools)   
MDiNE [MDiNE: a model to estimate differential co-occurrence networks in microbiome studies](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz824/5614428)   
MDPbiome [MDPbiome: microbiome engineering through prescriptive perturbations](https://academic.oup.com/bioinformatics/article/34/17/i838/5093255)    
microDecon [An R package for removing contamination from metabarcoding (e.g., microbiome) datasets post-sequencing](https://github.com/donaldtmcknight/microDecon)  
microbiotaPair [An R Package for Simplified Microbiome Analysis Workflows on Intervention Study](https://github.com/rusher321/microbiotaPair)    
MedTest [A Distance-Based Approach for Testing the Mediation Effect of the Human Microbiome](https://academic.oup.com/bioinformatics/article/34/11/1875/4810437)    
MegaR [An Interactive R Package for Metagenomic Sample Classification and Disease Prediction using Microbiota and Machine Learning](https://github.com/BioHPC/MegaR)     
MelonnPan [Model-based Genomically Informed High-dimensional Predictor of Microbial Community Metabolic Profiles](https://github.com/biobakery/melonnpan)  
Metacoder [An R package for visualization and manipulation of community taxonomic diversity data](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005404)   
metagenomeSeq [Differential abundance analysis for microbial marker-gene surveys](http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2658.html)   
MetaLonDA [METAgenomic LONgitudinal Differential Abundance method](https://github.com/aametwally/MetaLonDA)   
metamicrobiomeR [Analysis of Microbiome Relative Abundance Data using Zero Inflated Beta GAMLSS and Meta-Analysis Across Studies using Random Effects Model](https://github.com/cran/metamicrobiomeR)    
metaSPARSim [An R tool for 16S rRNA-gene sequencing count data simulation](https://gitlab.com/sysbiobig/metasparsim)   
microbiome R package  [Tools for microbiome analysis in R](https://github.com/microbiome/microbiome)    
MicrobiomeDDA [An Omnibus Test for Differential Distribution Analysis of Microbiome Sequencing Data](https://academic.oup.com/bioinformatics/article/34/4/643/4470360)   
MicrobiomeHD [A standardized database of human gut microbiome studies in health and disease *Case-Control*](http://www.biorxiv.org/content/early/2017/05/08/134031)   
microbiomeMarker [R package for microbiome biomarker discovery](https://github.com/yiluheihei/microbiomeMarker)   
MicrobiomeR [MicrobiomeR: An R Package for Simplified and Standardized Microbiome Analysis Workflows](https://microbiomer.vallenderlab.org/)   
microbiomeutilities [Extending and supporting package based on microbiome and phyloseq R package](https://microsud.github.io/microbiomeutilities/)   
MicrobiotaProcess [MicrobiotaProcess: an R package for analysis, visualization and biomarker discovery of microbiome](https://github.com/YuLab-SMU/MicrobiotaProcess)   
miLineage [A General Framework for Association Analysis of Microbial Communities on a Taxonomic Tree](https://medschool.vanderbilt.edu/tang-lab/software/miLineage)   
MINT [Multivariate INTegrative method](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1553-8)   
mixDIABLO [Data Integration Analysis for Biomarker discovery using Latent variable approaches for â€˜Omics studies](http://mixomics.org/mixdiablo/)   
mixMC [Multivariate Statistical Framework to Gain Insight into Microbial Communities](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0160169)   
MMinte [Methodology for the large-scale assessment of microbial metabolic interactions (MMinte) from 16S rDNA data](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1230-3)   
MTA [Microbial trend analysis (MTA) for common dynamic trend, group comparison and classification in longitudinal microbiome study](https://github.com/chanw0/MTA)    
MicrobiomeExplor [An R package for the analysis and visualization of microbial communities](https://doi.org/10.1093/bioinformatics/btaa838)   
MicroBVS [Dirichlet-tree multinomial regression models with Bayesian variable selection - an R package](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03640-0)    
MicroNiche [An R package for assessing microbial niche breadth and overlap from amplicon sequencing data](https://academic.oup.com/femsec/article-abstract/96/8/fiaa131/5863182?redirectedFrom=fulltext)   
microViz [An R package for for analysis and visualization of microbiome sequencing data](https://david-barnett.github.io/microViz/)

> N    
 
NMIT [Microbial Interdependence Association Test--a Non-parametric Microbial Interdependence Test](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5696116/#!po=75.0000)    
NetCoMi [Network construction and comparison for microbiome data in R](https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbaa290/6017455)   
NBZIMM [Negative binomial and zero-inflated mixed models, with application to microbiome/metagenomics data analysis](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-020-03803-z)    

> O    

OMiSA [Optimal Microbiome-based Survival Analysis (OMiSA)](https://github.com/hk1785/OMiSA)   

> P   

pathostat [Statistical Microbiome Analysis on metagenomics results from sequencing data samples](https://bioconductor.org/packages/release/bioc/html/PathoStat.html) - [support deprecated](https://github.com/mani2012/PathoStat/issues/48)
phylofactor [Phylogenetic factorization of compositional data](https://peerj.com/articles/2969/)   
phylogeo [Geographic analysis and visualization of microbiome data](https://www.ncbi.nlm.nih.gov/pubmed/25913208)   
Phyloseq [Import, share, and analyze microbiome census data using R](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0061217)   
Pldist [Pldist: Ecological Dissimilarities for Paired and Longitudinal Microbiome Association Analysis](https://academic.oup.com/bioinformatics/article-abstract/35/19/3567/5341424?redirectedFrom=fulltext)   
powmic [Power assessment in microbiome case-control studies](https://github.com/lichen-lab/powmic)   

> Q   

qgraph [Graph Plotting Methods, Psychometric Data Visualization and Graphical Model Estimation](https://cran.r-project.org/web/packages/qgraph/qgraph.pdf)   
qiime2R [Importing QIIME2 artifacts and associated data into R sessions](https://github.com/jbisanz/qiime2R)   
qiimer [R tools compliment qiime](https://github.com/kylebittinger/qiimer)    

> R    

RAM [R for Amplicon-Sequencing-Based Microbial-Ecology](https://cran.r-project.org/web/packages/RAM/RAM.pdf)   
RCM [A unified framework for unconstrained and constrained ordination of microbiome read count data](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0205474)   
RevEcoR [Reverse Ecology analysis in R](https://github.com/yiluheihei/RevEcoR)   
Rhea [A pipeline with modular R scripts](https://peerj.com/articles/2836/)   

> S   

selbal [Balances: a New Perspective for Microbiome Analysis](https://msystems.asm.org/content/3/4/e00053-18)  
ShinyPhyloseq [Web-tool with user interface for Phyloseq](http://joey711.github.io/shiny-phyloseq/)     
SIAMCAT [Statistical Inference of Associations between Microbial Communities And host phenoTypes](https://bioconductor.org/packages/release/bioc/html/SIAMCAT.html)   
SigTree [Identify and Visualize Significantly Responsive Branches in a Phylogenetic Tree](http://www.sciencedirect.com/science/article/pii/S2001037017300132)    
SparseMCMM [Estimating and testing the microbial causal mediation effect with the high-dimensional and compositional microbiome data (SparseMCMM)](https://sites.google.com/site/huilinli09/software)   
SPIEC-EASI [Sparse and Compositionally Robust Inference of Microbial Ecological Networks](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004226)   
SplinectomeR [SplinectomeR Enables Group Comparisons in Longitudinal Microbiome Studies](https://www.frontiersin.org/articles/10.3389/fmicb.2018.00785/full)   
StructFDR [False Discovery Rate Control Incorporating Phylogenetic Tree Increases Detection Power in Microbiome-Wide Multiple Testing](https://academic.oup.com/bioinformatics/article/33/18/2873/3824757)   
structSSI [Simultaneous and Selective Inference for Grouped or Hierarchically Structured Data](https://www.jstatsoft.org/article/view/v059i13)   

> T   

Tax4Fun [Predicting functional profiles from metagenomic 16S rRNA gene data](https://www.ncbi.nlm.nih.gov/pubmed/25957349)    
taxize [Taxonomic Information from Around the Web](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3901538/)    
TreeSummarizedExperiment [An extension class of SummarizedExperiment to allow hierarchical structure on the row or column dimension](https://github.com/fionarhuang/TreeSummarizedExperiment)   
themetagenomics [Exploring Thematic Structure and Predicted Functionality of 16S rRNA Amplicon Data](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0219235)    
tidyMicro [A pipeline for microbiome data analysis and visualization using the tidyverse in R](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-021-03967-2)  

> V   

vegan  [R package for community ecologists](https://github.com/vegandevs/vegan)   

> Y   

yingtools2 [Tools and functions for working with clinical and microbiome data](https://github.com/ying14/yingtools2)   

> Z    

zeroSum [Reference Point Insensitive Molecular Data Analysis](https://academic.oup.com/bioinformatics/article/33/2/219/2928229)   
ZIBBSeqDiscovery [A Zero-inflated Beta-binomial Model for Microbiome Data Analysis](https://onlinelibrary.wiley.com/doi/abs/10.1002/sta4.185)   

-------------------------------------------------------    

### Microbiome data sets  

1. HMP16SData [Efficient Access to the Human Microbiome Project through Bioconductor](https://bioconductor.org/packages/release/data/experiment/html/HMP16SData.html)   
2. HMP2Data [16s rRNA sequencing data from the Human Microbiome Project 2](https://bioconductor.org/packages/release/data/experiment/html/HMP2Data.html)  
3. microbiomeDataSets [Experiment Hub based microbiome datasets](https://bioconductor.org/packages/devel/data/experiment/html/microbiomeDataSets.html)   
4. curatedMetagenomicData [Accessible, curated metagenomic data through ExperimentHub](https://bioconductor.org/packages/release/data/experiment/html/curatedMetagenomicData.html)   

-------------------------------------------------------    
